openwisp_monitoring/monitoring/base/models.py

```diff
@@ -154,48 +156,97 @@ def _get_time(self, time):
             return parse_date(time)
         return time
 
-    def check_threshold(self, value, time=None, retention_policy=None, send_alert=True):
+    def _set_is_healthy(self, alert_settings, value):
         """
-        checks if the threshold is crossed
-        and notifies users accordingly
+        Sets the value of "is_healthy" field if "value"
+        crosses threshold defined in "alert_settings".
```

Comment:
We could add that this method does not take into account the alert settings tolerance, which is done instead by the other method (pointing out which one).